plugins/userdata/ElectronUserData.cc

```diff
@@ -175,11 +187,13 @@ void ElectronUserData::produce(edm::Event& iEvent, const edm::EventSetup& iSetup
 			const edm::Ptr<pat::Electron> elPtr(electrons, index);
 			std::vector < uint > idCutsToInvert { 99 };
 
+			// std::cout << "Energy Correction for electron : " << index << " is: " << electronSeedCorrections(iEvent, elPtr) << std::endl;
+			el.addUserFloat("energyCorrection", electronSeedCorrections(iEvent, elPtr));
```

Comment:
OK, so let's move it into an issue instead - in case one of us has 'spare' time :)